examples/components/MNIST/traininsilo/spec.yaml

```diff
@@ -43,6 +43,31 @@ inputs:
     description: batch size
     default: 64
     optional: true
+  dp:
+    type: boolean
+    description: differential privacy
+    default: false
+    optional: true
+  dp_epochs:
```

Comment:
@garg-amit dp_epoch, epoch (for a local train), and num_of_iterations are not independent from each other. dp_epoch=epoch* num_of_iterations. So in order to avoid potential user mistakes that they arbitrarily specify the dp_epoch, epoch and num_of_iteratios which doesn't satisfy the required relationship of dp_epoch=epoch* num_of_iterations, I suggest that we remove this dp_epoch input arg (and thus only have epoch and num_of_iterations inut arg), and compute dp_epoch using the relationship dp_epoch=epoch* num_of_iterations.

Reply:
makes sense! Unfortunately, the input parameter "num of iterations" is not available at this time (the "iteration num" field only specifies the current iteration number). I still believe it would be better to use `num of iteration` as an input parameter rather than `dp epochs` to prevent misunderstandings. I'll do the required changes

Reply:
done!